docs/topics/sb-features-and-frameworks-summary.adoc

```diff
@@ -20,10 +20,10 @@ a|
 Switches between services and continues to process incoming requests without interruption in case of service failure.
 
 a|
-Yes - using Kubernetes API
+Yes
 
 a|
-Spring Cloud Kubernetes - Hystrix
+link:https://github.com/spring-cloud-incubator/spring-cloud-kubernetes[Spring Cloud Kubernetes] - Hystrix
```

Comment:
Actually, this isn't using Spring Cloud Kubernetes, but rather Spring Cloud Netflix (https://github.com/spring-cloud/spring-cloud-netflix) Hystrix support (via the spring-cloud-starter-netflix-hystrix starter). Note that this starter has been renamed in more recent Spring Cloud version and this is the new name. Note also that we probably don't need to mention the actual starter for simplicity's sake.

Reply:
Ha, very good point! I totally missed that.